Formula/w/weaviate.rb

```diff
@@ -0,0 +1,30 @@
+class Weaviate < Formula
+  desc "Open-source vector database that stores both objects and vectors"
+  homepage "https://weaviate.io/developers/weaviate/"
+  url "https://github.com/weaviate/weaviate/archive/refs/tags/v1.27.6.tar.gz"
+  sha256 "d8cad0339ccfd081be6afe1afd879f64bba224ed7ecbdb8b974853b536584330"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s
+      -w
+      -X github.com/weaviate/weaviate/usecases/build.Version=#{version}
+      -X github.com/weaviate/weaviate/usecases/build.BuildUser=#{tap.user}
+      -X github.com/weaviate/weaviate/usecases/build.BuildDate=#{time.iso8601}
+    ]
+    system "go", "build", *std_go_args(ldflags:), "./cmd/weaviate-server"
+  end
+
+  test do
+    port = free_port
+    pid = fork do
+      system bin/"weaviate", "--host", "0.0.0.0", "--port", port, "--scheme", "http"
+    end
+    sleep 5
+    assert_match version.to_s, shell_output("curl localhost:#{port}/v1/meta")
+    Process.kill "TERM", pid
+  end
```

Comment:
```suggestion
  end

  test do
    port = free_port
    pid = spawn bin/"weaviate", "--host", "0.0.0.0", "--port", port.to_s, "--scheme", "http"
    sleep 5
    assert_match version.to_s, shell_output("curl localhost:#{port}/v1/meta")
  ensure
    Process.kill "TERM", pid
    Process.wait pid
  end
```

Reply:
Good point to use spawn and rely on ensure.
I updated the PR.

Thanks @daeho-ro